### ISSUE TITLE
Apache license mentioned in Theia-BackEnd/mvnw.cmd

### ISSUE THREAD

**Issue by phkrief** [eclipse-researchlabs/smartclide-security#5]
Dear 

The Apache license is mentioned in the file Theia-BackEnd/mvnw.cmd.
Do you plan to use this license on the repository knowing the LICENSE is missing or is it only copy/paste mistake?

Thx

**Comment by deanone** [comment]
Dear @phkrief ,

I will investigate it with my colleagues.

Best,

Athanasios

**Comment by deanone** [comment]
Dear @phkrief ,

Quick questions:

1. Given that we are going to have EPL as the license of the module, we need to remove any reference to other licenses (e.g., Apache License)? Does the reference of Apache License in the mvnw.cmd file create a conflict?
2. I saw that you have created a new branch called 'copyright' that includes the LICENSE file. Why don't you make a pull request for that? You can assign it to me

**Comment by phkrief** [comment]
Yes, it does. A repository should reference only one license, as far as I know
